.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,65 @@
+name: Build and test libslac
+on:
+  pull_request: {}
+  workflow_dispatch:
+    inputs:
+      runner:
+        description: Which runner to use
+        type: choice
+        default: 'ubuntu-22.04'
+        required: true
+        options:
+          - 'ubuntu-22.04'
+          - 'large-ubuntu-22.04-xxl'
+  schedule:
+    - cron: '37 13,1 * * *'
+
```

Comment:
Ditto wrt scheduled runs.
I wonder if there is a way to standardize/reuse this part of the workflow as well, so that we can ensure that all the workflows and invoked the same way.

@christopher-davis-afs is that something you can look up and see if there are best practices around?

Reply:
Removed the scheduled run. This was something inherited elsewhere, but it doesn't seem like it's being carried forward.